docs/articles_en/openvino-workflow/running-inference/optimize-inference/precision-control.rst

```diff
@@ -53,11 +75,33 @@ Code examples:
 Inference Precision
 ###################
 
-``ov::hint::inference_precision`` precision is a lower-level property that allows you to specify the exact precision the user wants, but is less portable. For example, CPU supports ``f32`` inference precision and ``bf16`` on some platforms, GPU supports ``f32`` and ``f16``, so if a user wants to an application that uses multiple devices, they have to handle all these combinations manually or let OV do it automatically by using higher level ``execution_mode`` property. Another thing is that ``inference_precision`` is also a hint, so the value provided is not guaranteed to be used by Runtime (mainly in cases where the current device does not have the required hardware capabilities).
+``ov::hint::inference_precision`` precision is a lower-level property that allows you
+to specify the exact precision the user wants, but is less portable. For example, CPU
+supports ``f32`` inference precision and ``bf16`` on some platforms, GPU supports ``f32``
+and ``f16``, so if a user wants to an application that uses multiple devices, they have
+to handle all these combinations manually or let OV do it automatically by using higher
+level ``execution_mode`` property. Another thing is that ``inference_precision`` is also
+a hint, so the value provided is not guaranteed to be used by Runtime (mainly in cases
+where the current device does not have the required hardware capabilities).
 
 .. note::
 
-   All devices only support floating-point data types (``f32``, ``f16``, ``bf16``) as a value for ``inference_precision`` attribute, because quantization cannot be done in Runtime.
+   All devices only support floating-point data types (``f32``, ``f16``, ``bf16``) as a value
+   for ``inference_precision`` attribute, because quantization cannot be done in Runtime.
+
+
+Limitation of the ``bf16`` inference precision
+++++++++++++++++++++++++++++++++++++++++++++++
+
+It is important to mention that inferring FP16 and FP32 LLM models with the ``bf16`` runtime
+precision may result in higher accuracy loss than the pre-determined threshold of 0.5%.
+Higher accuracy drop may occur when inferring original Pytorch models with ``bf16``, and is
+caused by a limited precision representation.
+
+To solve the issue, you might use an INT8 model and force the FP32 inference precision.
+The accuracy of an INT8 model with FP32 is nearly the same as of an FP16 model with ``bf16``.
+Additionally, selective FP32 execution of ops on CPU plugin together with the NNCF ``bf16``
+calibration could potentially mitigate the accuracy loss.
```

Comment:
this workaround would hurt performance  a lot on AMX-bf16 SPR platform for big batch size, in which case FMA is being used instead of AMX, in compute-bounded case(LLM batched inference/serving), this would reduced throughput by >70%, should we mention that?

Reply:
> this workaround would hurt performance a lot on AMX-bf16 SPR platform for big batch size, in which case FMA is being used instead of AMX, in compute-bounded case(LLM batched inference/serving), this would reduced throughput by >70%, should we mention that?

```suggestion
calibration could potentially mitigate the accuracy loss.

However, the solutions mentioned above would, unfortunately, also result in significant performance drop during a large batch size inference task on machines with Intel AMX-BF16 SPR. In such cases, the fused multiply-add operation (FMA) is used instead of AMX.
Also, in a compute-bound case, such as the LLM batch inference/serving, these work-arounds would drastically reduce the throughput by more than 70%. 

```

Reply:
I did a test locally again, base on the result I believe `by more than 60%` is more accurate